scripts/test/lld.py

```diff
@@ -18,36 +18,10 @@
 from support import run_command
 from shared import (
     fail, fail_with_error, files_with_pattern, options,
-    WASM_LINK_METADATA, WASM_EMSCRIPTEN_FINALIZE
+    WASM_EMSCRIPTEN_FINALIZE
```

Comment:
Does this fit on one line now?

Reply:
80 chars exactly... and PEP8 specifies <= 79